llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -6,6 +6,64 @@
 //
 //===----------------------------------------------------------------------===//
 
+/*
```

Comment:
Nice! How about making this into a Doxygen comment?
Here's an example: https://github.com/apple/llvm-project/blob/c3ebd6fb7d1fb49842fa9314183f8b97e411372e/lldb/source/Plugins/TypeSystem/Swift/DWARFImporterDelegate.cpp#L168